package.json

```diff
@@ -5,7 +5,8 @@
 	"author": "Nick Sweeting <archivebox-npm@sweeting.me>",
 	"license": "MIT",
 	"scripts": {
-		"archivebox": "./bin/archive"
+   "setup": "node -e \"const {execSync} = require('child_process'); Object.entries(JSON.parse(fs.readFileSync('package.json')).dependencies).forEach(globaldep => execSync('npm i -g ' + globaldep[1]))\"",
```

Comment:
I would remove this setup method, if they're viewing this `package.json` file then they're already in a repo dir where they can just run `npm install` rather than installing it system-wide. (as docker and github actions do)

Reply:
Sorry, forgot that the users normally don't clone the repository. Still, system wide should be better for users. I started using the docker version when readability and singlefile were mandatory because it was a pain to set the variables every time. If we ask the user to install them locally, we will need to add instructions to set `SINGLEFILE_BINARY` and `READABILITY_BINARY` as part of the installation, which just feels weird. Another option (I asked somewhere else I think) is to setup the binary by default to `node_modules/.bin/readability-extractor`.